webrtc-stats.html

```diff
@@ -768,6 +768,7 @@
              double             burstDiscardRate;
              double             gapLossRate;
              double             gapDiscardRate;
+             DOMHighResTimeStamp  lastPacketReceivedTimestamp;
```

Comment:
You're changing `RTCReceivedRTPStreamStats` here, which is the base of both `RTCInboundRTPStreamStats` and `RTCRemoteInboundRTPStreamStats`. In the latter case, the timestamps will not be based on the local clock of the reader, which might warrant a warning, so people don't try to compare it to other timestamps in the report.